sycl/test/regression/msvc_crt.cpp

```diff
@@ -0,0 +1,46 @@
+// RUN: %clang_cl -fsycl /MD -o %t1.exe %s
```

Comment:
```suggestion
// RUN: %clang_cl -fsycl -fsycl-targets=spir64-unknown-windows-sycldevice /MD -o %t1.exe %s
```
could be a w/a for the unrelated Windows failures.